src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -176,6 +176,7 @@ int sd_dhcp_server_new(sd_dhcp_server **ret, int ifindex) {
         server->n_ref = 1;
         server->fd_raw = -1;
         server->fd = -1;
+        server->fd_broadcast = -1;
```

Comment:
bind_to_interface should defualt to true here, too

Reply:
Thanks, I missed this one. All done